### ISSUE TITLE
Improved notebook for TIA publication

### ISSUE THREAD

**Issue by TomFer97** [SPSUnipi/EnergyCommunity.jl#13]
We could modify the folder run_cloud in order to provide useful materials for future users. I've opened a branch locally and tomorrow I'll make the pull request

**Comment by davide-f** [comment]
I agree, however, the material in run_cloud was intended more for internal purposes. In the future, we could delete that folder or repurpose it.
The main documentation should be available in examples and doc.
However, some selected material in run_cloud may be repurposed for examples later on indeed
